srcpkgs/dtkwidget/patches/01.patch

```diff
@@ -0,0 +1,323 @@
+diff --git a/debian/libdtkwidget2.symbols debian/libdtkwidget2.symbols
```

Comment:
is this debian stuff really necessary ?

Reply:
Actually no, But this are just the commit from upstream and I was to lazy to remove unnecessary stuff. If you want I can remove it.